easybuild/easyblocks/t/trinity.py

```diff
@@ -312,24 +315,53 @@ def sanity_check_step(self):
         """Custom sanity check for Trinity."""
 
         version = LooseVersion(self.version)
-        if version >= LooseVersion('2.0') and version < LooseVersion('2.3'):
-            sep = '-'
-        elif version >= LooseVersion('2.3') and version < LooseVersion('3.0'):
-            sep = "-Trinity-v"
+        if ((version >= LooseVersion('2.0') and version < LooseVersion('2.3')) or
+           (version >= LooseVersion('2.9') and version < LooseVersion('3.0'))):
+            sep = '-v'
+        elif version >= LooseVersion('2.3') and version < LooseVersion('2.9'):
+            sep = '-Trinity-v'
         else:
             sep = '_r'
-
-        if version >= LooseVersion('2.8') and version < LooseVersion('2000'):
-            chrysalis_bin = 'Chrysalis/bin/Chrysalis'
+        # Chrysalis
+        if version >= LooseVersion('2.9') and version < LooseVersion('2000'):
+            chrysalis_bin = 'Chrysalis/bin'
+            chrysalis_files = ['BubbleUpClustering',
+                               'CreateIwormFastaBundle',
+                               'QuantifyGraph',
+                               'Chrysalis',
+                               'GraphFromFasta',
+                               'ReadsToTranscripts']
+        elif version >= LooseVersion('2.8') and version < LooseVersion('2.9'):
+            chrysalis_bin = 'Chrysalis/bin'
+            chrysalis_files = ['Chrysalis']
+        else:
+            chrysalis_bin = 'Chrysalis'
+            chrysalis_files = ['Chrysalis']
+        chrysalis_bin_files = [os.path.join(chrysalis_bin, x) for x in chrysalis_files]
+
+        # Inchworm
+        if version >= LooseVersion('2.9') and version < LooseVersion('2000'):
+            inchworm_bin = 'Inchworm/bin' 
+            inchworm_files = ['FastaToDeBruijn',
+                              'fastaToKmerCoverageStats',
+                              'inchworm']
         else:
-            chrysalis_bin = 'Chrysalis/Chrysalis'
+            inchworm_bin = 'Inchworm/bin'
+            inchworm_files = ['inchworm']
+        inchworm_bin_files = [os.path.join(inchworm_bin, x) for x in inchworm_files]
```

Comment:
This can be simplified a bit:

```python
inchworm_bin = os.path.join('Inchworm', 'bin')
inchworm_files = ['inchworm']
if version >= LooseVersion('2.9') and version < LooseVersion('2000'):
    inchworm_files.extend(['FastaToDeBruijn', 'fastaToKmerCoverageStats'])
inchworm_bin_files = [os.path.join(inchworm_bin, x) for x in inchworm_files]

Reply:
👍 Thx @boegel for advice. Fixed